app/src/main/java/org/piwigo/ui/login/LoginActivity.java

```diff
@@ -76,13 +77,18 @@
     private AccountAuthenticatorResponse authenticatorResponse;
     private Bundle resultBundle;
 
+    public CountingIdlingResource idleResource;
+
     @Override
     @SuppressLint("ClickableViewAccessibility") // We are silencing the warning since we don't want to create an extra view for only one purpose..
     protected void onCreate(Bundle savedInstanceState) {
         AndroidInjection.inject(this);
         setTheme(R.style.Theme_Piwigo_Login);
         super.onCreate(savedInstanceState);
 
+        idleResource = new CountingIdlingResource("LoginActivity", true);
+        userManager.setIdleResource(idleResource);
```

Comment:
ah ok, the idling resource from here IS incremented by the moreBusy... mh. - I don't yet catch it...

Reply:
this was just WIP, no wonder you are confused - I tried 3 different approaches :)